src/components/CoordinatorHome.js

```diff
@@ -4,149 +4,209 @@ import styled from "styled-components"
+const CoordinatorHome = observer(({ store }) => {
 
-    <Table
-      data={store.coordinator.patients}
-      columns={[
+  const tableColumns= [
+    {
+      Header: "Participant Info",
+      columns: [
         {
-          Header: "medication_report_dates",
+          Header: "Status",
           accessor: "medication_report_dates",
-
-          Cell: patient => {
-            // TODO placeholder,
-            // until we get the data hooked up for this.
-            let random = Math.random()
-            let randomHealth =
-              random > 0.3 ?  ( random > 0.6 ?  "good" : "okay") : "bad"
-            let healthBadge = Icons[randomHealth]
-
-            return (
-              healthBadge
-            )
-          }
+          width: 110,
+          // Link this up to the patient so that it changes after a test result is submitted
+          Cell: row => (
+            <span>
+              <span style={{
+                color: row.value === 'notreported' ? '#ff2e00'
+                  : row.value === 'reported' ? '#ffbf00'
+                  : '#57d500',
+                transition: 'all .3s ease'
+              }}>
+                &#x25cf;
+              </span> {
+                row.value === 'notreported' ? 'Not Reported'
+                : row.value === 'reported' ? `Reported`
+                : 'Confirmed'
+              }
+            </span>
+          )
         },
-
-        { Header: "id", accessor: "id" },
-        { Header: "name", accessor: "name" },
-        { Header: "treatment_start_date", accessor: "treatment_start_date" },
-        { Header: "side_effects", accessor: "side_effects" },
-        { Header: "percent_reported", accessor: "percent_reported" },
-        { Header: "photo", accessor: "photo" },
-        { Header: "note", accessor: "note" },
-        { Header: "coordinator_note", accessor: "coordinator_note" },
-      ]}
-      defaultPageSize={5}
-    />
-  </Layout>
-))
-
+        {
+          Header: "First",
+          accessor: "firstname"
+        },
+        {
+          Header: "Last",
+          accessor: "lastname"
+        }
+      ]
+    },
+    {
+      Header: 'Reported Data',
+      columns: [
+        {
+          // TODO: Link up to the correct value
+          Header: "Took Medication",
+          accessor: "took_medication"
+        },
+        {
+          // TODO: put side effects into a line break list
+          Header: "Side Effects",
+          accessor: "side_effects",
+          Cell: e=> e.value.join("\n \n")
+        },
+        {
+          Header: "Photo",
+          accessor: "photo",
+          Cell: e=>
+            <PhotoPopout>
+              <Image
+                    src={strip_report}
+                    alt={"Strip report"}
+                    onClick={(e) => {
+                      e.preventDefault()
+                      e.stopPropagation()
+                    }}
+              />
+            </PhotoPopout>
```

Comment:
Ah, this could be the re-usable `src/primitives/PhotoPopout.js`.
Might also be a good place to try out Reakit theming, if that applies.